.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Build medic-couch2pg and run unit test cases
+
+on: [ push, pull_request ]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [ 8, 10, 12 ]
+
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.CI_DOCKER_USERNAME }}
+          password: ${{ secrets.CI_DOCKER_PASSWORD }}
+
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: docker-compose build --build-arg node_version=${{ matrix.node-version }} test
+        env:
+          COUCH2PG_SLEEP_MINS: 120
+          COUCH2PG_DOC_LIMIT: 1000
+          COUCH2PG_RETRY_COUNT: 5
```

Comment:
These environment variables look new - are they needed? What changed?

Reply:
These were added in this [commit](https://github.com/medic/medic-couch2pg/commit/471d62f36425aa80385c0c193aa379e68f22e5ea#diff-cbf01e88a71952667baadd3a091707e87cf2e42b0c4cff1d2569af68e134e1fe) (the [README](https://github.com/medic/medic-couch2pg/blob/master/README.md)) and used for supporting some logic around the [runner](https://github.com/medic/medic-couch2pg/blob/1bbab3e3014504eb1699f1e5317d6abf73b8bfb9/libs/medic/runner.js#L55) and the [inquirer](https://github.com/medic/medic-couch2pg/blob/1bbab3e3014504eb1699f1e5317d6abf73b8bfb9/inquirer.js#L63). 
Maybe we can leave it to mirror prod, is that okay?

Reply:
I can see it runs without these env variables in local.

But asking around, I can confirm it's used in production where:
COUCH2PG_SLEEP_MINS = is the number of minutes between syncs
COUCH2PG_DOC_LIMIT = is the number of documents couch2pg fetches from couchdb everytime it calls couchdb
COUCH2PG_RETRY_COUNT =  is how many times couch2pg should keep trying syncing docs after experiencing an error.

Reply:
What I meant was, specifying the environment variables in the build is new. I can't see them in the travis file anywhere. Why did you add them here? Are they needed?

Reply:
Removed the env variables.